src/services/doppler-api-client.ts

```diff
@@ -11,6 +11,7 @@ export interface DopplerApiClient {
     email: string,
     apikey: string,
   ): Promise<ResultWithoutExpectedErrors<CampaignDeliveryCollection>>;
+  getSubscribers(email: string): Promise<ResultWithoutExpectedErrors<SubscriberCollection>>;
```

Comment:
Remember here instead of `email` we are using a searching text that can be part of: email, last name or name.

Reply:
ohh you right! thanks